marvel_superheroes/src/components/HomePage/HomePage.tsx

```diff
@@ -70,10 +75,11 @@ class HomePage extends React.Component<IProps, IState> {
     }
 
     searchBarHandleSubmit = (changeInputState: boolean, nameStartsWith: string, changeSelectState: boolean, orderBy: string): void => {
+        const queryIsExist = queryString.parse(this.props.location.search).query;
+        const sortIsExist = queryString.parse(this.props.location.search).sort;
```

Comment:
you can make it once queryString.parse(this.props.location.search). no need to repeat this